server/xpub/entities/manuscript/resolvers.test.data.js

```diff
@@ -0,0 +1,77 @@
+exports.userData = {
+  username: 'testuser',
+  email: 'test@example.com',
+  orcid: '0000-0003-3146-0256',
+  oauth: { accessToken: 'f7617529-f46a-40b1-99f4-4181859783ca' },
+}
+
+exports.badUserData = {
+  username: 'baduser',
+  email: 'bad@example.com',
+  orcid: '0000-0001-0000-0000',
+  oauth: { accessToken: 'badbadbad-f46a-40b1-99f4-4181859783ca' },
+}
+
+exports.manuscriptInput = {
+  meta: {
+    title: 'My manuscript',
+    articleType: 'research-article',
+    subjects: ['cancer-biology'],
+  },
+  suggestedSeniorEditors: ['ab12', 'cd34'],
+  opposedSeniorEditors: ['ij90'],
+  opposedSeniorEditorsReason: 'I do not like green eggs and ham',
+  suggestedReviewingEditors: ['ef56', 'gh78'],
+  opposedReviewingEditors: ['kl12'],
+  opposedReviewingEditorsReason: 'I do not like them, Sam I am',
+  suggestedReviewers: [
+    { name: 'Reviewer 1', email: 'reviewer1@mail.com' },
+    { name: 'Reviewer 2', email: 'reviewer2@mail.com' },
+    { name: 'Reviewer 3', email: 'reviewer3@mail.com' },
+  ],
+  opposedReviewers: [{ name: 'Reviewer 4', email: 'reviewer4@mail.com' }],
+  opposedReviewersReason: 'Reviewer 4 is my wife',
+  suggestionsConflict: false,
+  coverLetter: 'my cover letter',
+  author: {
+    firstName: 'Firstname',
+    lastName: 'Lastname',
+    email: 'mymail@mail.com',
+    aff: 'Institution Inc',
+  },
+  previouslyDiscussed: 'Discussed this with bob',
+  previouslySubmitted: ['An important finding about ants'],
+  cosubmission: ['More interesting observations on chickens'],
+  submitterSignature: 'Sneha Berry',
+  disclosureConsent: true,
+}
+
+exports.manuscriptMatch = {
```

Comment:
Is there any reason why all of the keys in this object are in a different order? It might be a little simpler to follow the logic if it were in the same order as the object above?

Could we maybe call this `expectedManuscript` or `expectedManuscriptResult` or something similar? I was kind of expecting everything in this file to be data that we were testing with, as opposed to also being the results that we expect to be returned. The name might help with that? And then the test would read
`expect(actualManuscript).toMatchObject(expectedManuscript)`

Reply:
That's just the way they came out. Makes sense to reorder and rename.